atomic_reactor/plugins/pre_reactor_config.py

```diff
@@ -50,26 +50,28 @@ def __init__(self, name, max_concurrent_builds, enabled=True):
         self.enabled = enabled
 
 
+class ReactorConfigKeys(object):
```

Comment:
I assume the intention is to be able to use this in other/upcoming plugins. In this case, maybe a one-line doc-string would be nice - what is this, and what is the intention (even though at this point this is pretty obvious).